### PR TITLE
Adapter view holder

### PR DESCRIPTION
**Pull request by piperwhite**

### CODE REVIEW

File: app/src/main/java/com/whiteandc/myfabrics/FabricListActivity.java [hunk 1]
```diff
@@ -5,21 +5,42 @@
...
+    }
+
+    @Override
+    public void showFabrics(List<Fabric> fabrics) {
+        adapter.addFabrics(fabrics);
+        adapter.notifyDataSetChanged();
+    }
+
+    @Override
+    protected void onDestroy() {
```
Comment (CPalacin) [inline app/src/main/java/com/whiteandc/myfabrics/FabricListActivity.java:43]: Add `presenter.detach()`

**Review by CPalacin — COMMENTED** [review]
One small change
